project/iterative_sorting.py

```diff
@@ -1,4 +1,7 @@
 # Complete the selection_sort() function below in class with your instructor
+# init
+
+
```

Comment:
nice, looks clean. for the swap below, there's a cool feature in Python. you can do

arr[cur_index], arr[smallest_index] = arr[smallest_index], arr[cur_index]